src/main/java/frc/robot/RobotContainer.java

```diff
@@ -51,11 +54,13 @@ public class RobotContainer {
   private final ShuffleboardTab m_autoTab = Shuffleboard.getTab("Auto");
   private final ShuffleboardTab m_controllerTab = Shuffleboard.getTab("Controller");
   private final ShuffleboardTab m_testTab = Shuffleboard.getTab("Test");
+  private final ShuffleboardTab m_elevatorTab = Shuffleboard.getTab("Elevator");
 
   // The robot's subsystems are defined here...
-  private final Drivetrain m_drive;
-  private final FourBarArm m_arm;
-  private final Intake m_intake;
+  private Drivetrain m_drive;
+  private FourBarArm m_arm;
+  private Intake m_intake;
+  private Elevator m_elevator;
```

Comment:
Not sure why these are not final

Reply:
If I make elevator final it gives me the error: The blank final field m_elevator may not have been initialized